src/pages/GraphQL.js

```diff
@@ -103,13 +103,15 @@ GraphQL has a rapidly growing community. To stay up-to-date about everything tha
 
 * [GraphQL Weekly](https://www.graphqlweekly.com/): Weekly newsletter about GraphQL
 * [GraphQL Radio](https://www.graphqlradio.com/): Podcast discussing real-world use cases of GraphQL
+* [GraphQL Europe](https://www.graphql-europe.org/): Europe's biggest GraphQL conference
+* [Prisma blog](https://blog.graph.cool/): Technical deep dives and tutorials all around GraphQL development
 
 For an in-depth learning experience, visit the [How to GraphQL](https://www.howtographql.com/) fullstack tutorial website.
 
 
 ## Minimal Example
 
-Here is a minimal example that demonstrates how to load data using [Graphcool](https://www.graph.cool/) and [Apollo Client](http://dev.apollodata.com/react/). If you want to understand more about how it works, check out the following chapters.
+Here is a minimal example that demonstrates how to load data using [Prisma](https://www.graph.cool/), [\`graphql-yoga\`](https://github.com/graphcool/graphql-yoga) and [Apollo Client](https://www.apollographql.com/client). If you want to understand more about how it works, check out the following chapters.
```

Comment:
Should this Prisma link point to a different website? Looks like it still points to graph.cool

Reply:
Ah thanks, great catch! I updated the link to the official Prisma page.